src/xml_dump.c

```diff
@@ -195,6 +195,10 @@ cr_xml_dump_files(xmlNodePtr node, cr_Package *package, int primary)
         if (entry->type && entry->type[0] != '\0' && strcmp(entry->type, "file")) {
             cr_xmlNewProp(file_node, BAD_CAST "type", BAD_CAST entry->type);
         }
+
+        if (filelists_ext && entry->digest && entry->digest[0] != '\0') {
+            cr_xmlNewProp(file_node, BAD_CAST "hash", BAD_CAST entry->digest);
```

Comment:
This probably should have an attribute to distinguish the digest algorithm used alongside the digest itself.

Reply:
There is a new XML field for each package, that indicate the algorithm (`<checksum/>`), this tag is the file digest itself